app.py

```diff
@@ -96,7 +96,7 @@ async def submit_form(form: SubmitIn,
                      redirect_url=redirect_url)
 
 
-@app.post('/api/interceptum-post', response_model=SubmitOut)
+@app.post('/webhook/interceptum-post', response_model=SubmitOut)
```

Comment:
Let's make this `'webhook/interceptum-post/'` in order to make it inline with the rest of our endpoints (even though I'm pretty sure it's wrong to put the / at the end)